modules/apps/29-fee/ibc_module.go

```diff
@@ -221,7 +221,7 @@ func (im IBCModule) OnAcknowledgementPacket(
 	acknowledgement []byte,
 	relayer sdk.AccAddress,
 ) error {
-	if !im.keeper.IsFeeEnabled(ctx, packet.SourcePort, packet.SourceChannel) {
+	if !im.keeper.IsFeeEnabled(ctx, packet.SourcePort, packet.SourceChannel) || im.keeper.IsLocked(ctx) {
```

Comment:
This will fail because the counterparty will still be sending an incentivized ack. We should unmarshal the ack and just send to underlying without distributing fees